server/data/apps.json

```
@@ -1,4 +1,21 @@
 [
+  {
+    "appName": "Audicy",
+    "appType": [
+      "Discord Audio Streaming Bot"
+    ],
+    "appUrl": "https://audicy.xyz",
+    "appIconUrl": "audicylogo.png",
+    "platforms": [
+      "Discord"
+    ],
+    "supportedElements": [
+      {
+        "elementName": "Discord Server",
+        "elementURL": "https://discord.gg/4Vxsf4Y8KK"
```

I would recommend `Search` here along with a link to a page/blog that says it's being used:

```suggestion
        "elementName": "Search",
        "elementURL": "https://audicy.xyz/about"
```